src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let discount = 0;
```

Comment:
<img width="1056" alt="Screenshot 2023-11-11 at 17 57 16" src="https://github.com/mate-academy/js_task-transportation-on-vacation/assets/135232001/08848025-4af3-48fb-a401-441dc8fcab9a">
Use the checklist for this task to create constants and name them